### PR TITLE
VEC-317 0.9.1 support

### PR DESCRIPTION
**Pull request by jdogmcsteezy**
- change --hnsw-max-records to --hnsw-m
- change --hnsw-healer-schedule-delay to --hnsw-healer-schedule
- change --hnsw-merge-parallelism to --hnsw-merge-index-parallelism
- add --hnsw-merge-reindex-parallelism

### CODE REVIEW

**Comment by jdogmcsteezy** [comment]
Can be review now. Small changes to the docker compose files will be needed once AVS 0.9.1 is released and changes to the go-client are merged

File: cmd/flags/hnsw.go [hunk 1]
```diff
@@ -73,18 +73,18 @@ func NewHnswHealerFlags() *HealerFlags {
...
-	flagSet.Var(&cf.ScheduleDelay, HnswHealerScheduleDelay, "The time delay between the termination of a healer run and the commencement of the next one for an index.") //nolint:lll // For readability
-	flagSet.Var(&cf.Parallelism, HnswHealerParallelism, "Maximum number of records to heal in parallel.")                                                                //nolint:lll // For readability
+	flagSet.Var(&cf.MaxScanRatePerNode, HnswHealerMaxScanRatePerNode, "Maximum allowed record scan rate per AVS node.")                         //nolint:lll // For readability
+	flagSet.Var(&cf.MaxScanPageSize, HnswHealerMaxScanPageSize, "Maximum number of records in a single scanned page.")                          //nolint:lll // For readability
+	flagSet.Var(&cf.ReindexPercent, HnswHealerReindexPercent, "Percentage of good records randomly selected for reindexing in a healer cycle.") //nolint:lll // For readability
+	flagSet.Var(&cf.Schedule, HnswHealerSchedule, "The quartz cron expression defining schedule at which the healer cycle is invoked.")         //nolint:lll // For readability
```
Comment (dwelch-spike) [inline cmd/flags/hnsw.go:86]:
```suggestion
	flagSet.Var(&cf.Schedule, HnswHealerSchedule, "The quartz cron expression defining the schedule at which the index healer cycle is invoked.")         //nolint:lll // For readability
```
nit

**Review by dwelch-spike — APPROVED** [review]
LGTM, one doc string nitpick

File: bin/asvecrpm/asvec.spec [hunk 2]
```diff
@@ -12,11 +12,16 @@ Group: aerospike
...
 /opt/aerospike/bin/asvec
+/usr/bin/asvec
+
+%install
+# Ensure the buildroot directories exist
+mkdir -p %{buildroot}/opt/aerospike/bin
+mkdir -p %{buildroot}/usr/bin
 
 %prep
 ln -sf /opt/aerospike/bin/asvec %{buildroot}/usr/bin/asvec
```
Comment (dwelch-spike) [inline bin/asvecrpm/asvec.spec:27]: Should this be... ?
```suggestion
ln -sf %{buildroot}/opt/aerospike/bin/asvec %{buildroot}/usr/bin/asvec
```

**Review by dwelch-spike — COMMENTED** [review]
Double checking one thing

File: bin/asvecrpm/asvec.spec (hunk 2 above)
Comment (jdogmcsteezy) [reply bin/asvecrpm/asvec.spec:27]: no, the first arg does not need to exist. We are creating a link from the second argument to the first. The second argument can move around but the first argument is constant

**Pull request merged**